x-pack/solutions/search/plugins/search_indices/public/components/index_documents/recent_docs_action_message.tsx

```diff
@@ -10,8 +10,7 @@ import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiPanel } from '@elastic/eui';
 import { useKibana } from '../../hooks/use_kibana';
-
-import { DEFAULT_PAGE_SIZE } from './constants';
+import { DEFAULT_DOCUMENT_PAGE_SIZE } from '../../constants';
```

Comment:
updated this to use that same constant as the hook. That way when we change one this component will match.